packages/shared/src/utils/routes/routes.js

```diff
@@ -8,6 +8,8 @@ export const routes = {
     proof_of_identity: '/account/proof-of-identity',
     proof_of_address: '/account/proof-of-address',
     deriv_password: '/account/deriv-password',
+    deactivate_account: '/account/deactivate-account',
+    account_deactivated: '/account-deactivated',
```

Comment:
Why do we need a route for this?

Reply:
redirect user to a new page and (based on the design) and logout him and show a message about you your account is deactivated.